vite.config.js

```diff
@@ -1,8 +1,41 @@
+/* eslint-disable no-undef */
+import * as path from 'path';
+
+/** @type {import('vite').UserConfig} */
 export default {
-    base: '/',
-    server: {
-      port: 5173,
-      hot: true,
-      host: '127.0.0.1'
+  // base: '/',
+  build: {
+    outDir: path.resolve(__dirname, '/dist'),
```

Comment:
this needs to be 

```suggestion
    outDir: path.resolve(__dirname, './dist'),

```

so that it stays relative to the root dir